app/components/PageTitle.js

```diff
@@ -1,15 +1,20 @@
 // @flow
 import * as React from "react";
 import { Helmet } from "react-helmet";
+import AuthStore from "stores/AuthStore";
 
 type Props = {
   title: string,
   favicon?: string,
+  auth: AuthStore,
```

Comment:
mobx `inject` and `observer` needs to be used to make this store available to the component.